cla.md

```diff
@@ -42,7 +42,7 @@ This section assures project stewards that my contributions come under acceptabl
 
 When I make contributions including others' work:
 
-1.  I promise to take basic steps to make sure that work is licensed under permissive open license terms that Blue Oak Council rates "bronze" or better at <https://blueoakcouncil.org/list>.
+1.  I promise to take basic steps to make sure that work is licensed under permissive open license terms that Blue Oak Council rates "bronze" or better at <https://blueoakcouncil.org/list>, and to mention in my pull requests when it is not.
```

Comment:
This addition will probably be read to imply that the agreement could (as opposed to couldn't, which is how I had read it before the change) be used for non-permissive projects that are OK with accepting contributions that include third party material under a compatible but non-permissive (e.g, the same) license. I still probably wouldn't be thrilled with it if I were the steward of a non-permissive project, but on the other hand contributions that include third party material are probably rare, so I guess it shouldn't be a big deal.

For what it's worth, the addition feels slightly redundant with the following point.

Reply:
@mlinksva I've just pushed a new commit that elevates the requirement from "mentioning" non-permissive work to making a PR comment about it. There is some redundancy with the idea of presenting good licensing records that the steward can review. I think the point here is making sure non-permissive work gets brought to their attention more affirmatively.

Reply:
Makes sense!